server/core/kv.go

```diff
@@ -114,6 +114,11 @@ func (kv *KV) SaveStore(store *metapb.Store) error {
 	return saveProto(kv.KVBase, kv.storePath(store.GetId()), store)
 }
 
+// DeleteStore delete one store from KV.
```

Comment:
s/delete/deletes/g

Reply:
Next time I will pay attention to it. :grin: